content/dev-tools/dev-tools.md

```diff
@@ -50,10 +50,6 @@ The following tools are maintained by the XRP Ledger community:
 
     See all [tokens issued](issued-currencies.html) in the XRP Ledger and use preset tools to issue custom tokens at the click of a button.
 
-- **[XRP Stats](https://ledger.exposed/)**
-
-    View lists of who has the most XRP in their possession or locked up in [escrow](escrow.html).
-
 - **[XRPCharts](https://xrpcharts.ripple.com/)**
```

Comment:
The XRPCharts site states that it is deprecated and redirects users to the XRPL Explorer. Suggest adding a deprecation note here or removing this link.